spring-cloud-deployer-resource-maven/src/main/java/org/springframework/cloud/deployer/resource/maven/MavenArtifactResolver.java

```diff
@@ -58,9 +58,10 @@
 import org.springframework.util.StringUtils;
 
 /**
- * Resolves a {@link MavenResource} using <a href="http://www.eclipse.org/aether/>aether</a> to
- * locate the artifact (uber jar) in a local Maven repository, downloading the latest update from a
- * remote repository if necessary.
+ * Resolves a {@link MavenResource} using <a
```

Comment:
Did you change something here? Hard to tell with the reformatting

Reply:
Sorry, no changes here.